examples/example.rs

```diff
@@ -0,0 +1,127 @@
+extern crate neat_gru;
+
+use itertools::Itertools;
+use neat_gru::game::Game;
+use neat_gru::neural_network::nn::NeuralNetwork;
+use neat_gru::topology::topology::Topology;
+use neat_gru::train::train::Train;
+struct Player {
+    pub net: NeuralNetwork<f64>,
+}
+
+impl Player {
+    pub fn new(net: NeuralNetwork<f64>) -> Player {
+        Player { net }
+    }
+    /// Runs all the inputs and calculates the outputs
+    fn run(&mut self) -> f64 {
+        // Get the inputs
+        let inputs = XOR::get_inputs();
+        // Calculate a score for every input
+        let outputs: Vec<f64> = inputs.iter().map(|i| self.net.compute(i)[0]).collect();
+        let mut scores: Vec<f64> = vec![];
+        for idx in 0..(inputs.len() - 1) {
+            scores.push(compute_score(&inputs[idx], outputs[idx]));
```

Comment:
You need to round the output because it will never be exactly `1` from a `NN` output

Reply:
You could also consider having 2 outputs and considering the first one as 0 and the second one as 1, then consider the output as `index(max(output[0], output[1]))`